Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.Shared/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
```

Comment:
If not 4.6.2, then minimum supported version

Reply:
This project was an appendage, and has been removed. Code is shared by linked files in a shared folder.